src-data/witan/models/upload_data.clj

```diff
@@ -0,0 +1,31 @@
+(ns witan.models.upload-data
+  (:require [clojure.string :as str]
+            [amazonica.aws.s3 :as aws]
+            [amazonica.aws.s3transfer :as aws-s3])
+  (:gen-class))
+
+(def default-bucket    "witan-data")
+(def default-profile   "witan")
```

Comment:
Not sure about the use of profiles here, will these get created on the box on start up? Normally just supply the region and have amazonica pick up the creds from the evironment